persist/migrations/000002_pubkey_usernames.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE public.lnurl_pubkey_usernames (
```

Comment:
If I understand correctly these are in a separate table, because a pubkey can have many webhooks, but a pubkey can only have a single username?

Reply:
The reason for the separate table was because the pubkey/webhook table can be cleaned where all entries for a pubkey could be removed, this prevents someone sniping a username from them in that case.

Not sure if I see a valid case for multiple usernames per pubkey... although I guess it could help target to a specific device / app (not the last registration).